extensions/hints/src/use-flow-page.tsx

```diff
@@ -0,0 +1,171 @@
+  "google-calendar": "../assets/GoogleCalendar.svg",
+  clickup: "../assets/ClickUp.svg",
+  hubspot: "../assets/HubSpot.svg",
+  notion: "../assets/Notion.svg",
+  pipedrive: "../assets/Pipedrive.svg",
+  trello: "../assets/Trello.svg",
+  jira: "../assets/Jira.svg",
+} as any;
+
+export default function Command() {
+  const preferences = getPreferenceValues<Preferences>();
+  const [pages, setPages] = useState([]);
+  const [hashtag, setHashtag] = useState([]);
+  const [searchText, setSearchText] = useState<string>("");
+  const [isLoading, setIsLoading] = useState(true);
+
+  async function search() {
+    const toast = await showToast({ title: "Updating list of integrations...", style: "ANIMATED" as any });
+
+    const response = await fetch(
+      `https://europe-west3-slipbox-6f705.cloudfunctions.net/getUserFlows?api_key=${preferences.api_token}`
+    );
+    const data = await response.json();
+    setPages(data as any);
+    toast.style = Toast.Style.Success;
+    toast.title = "List of integrations updated";
+  }
+
+  useEffect(() => {
+    const searchFlows = async () => {
+      setIsLoading(true);
+      await search();
+      setIsLoading(false);
+    };
+    searchFlows();
+  }, []);
+
+  useEffect(() => {
+    const croppedHashtag: any = searchText.split(/[\s\n\r]/gim).filter((v) => v.startsWith("#"));
+    setHashtag(croppedHashtag[0]);
+  }, [searchText]);
+
+  function hasRequiredError(searchText: string) {
+    return !searchText;
+  }
+
+  async function handleSubmit(searchText: string) {
+    if (hasRequiredError(searchText)) return;
+
+    console.log(searchText);
+
+    const toast = await showToast({ title: "Sending to Hints...", style: "ANIMATED" as any });
+
+    fetch(
+      `https://europe-west3-slipbox-6f705.cloudfunctions.net/createHintsFromRaycast?api_key=${preferences.api_token}`,
+      {
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify({ content: searchText }),
+      }
+    );
+    toast.style = Toast.Style.Success;
+    toast.title = "Successfully sent";
+  }
+
+  // ToDo add custom filtering to filter list of flows
+  // filterList(items.filter((item) => item.includes(searchText)));
+
+  function handleSubmitWithTag(flowHashtag: string) {
+    const messgaeWithHashtag = `${searchText} #${flowHashtag}`;
+    console.log(messgaeWithHashtag);
+    handleSubmit(messgaeWithHashtag);
+    setSearchText(``);
+  }
+
+  async function handleSearchFromClipboard() {
+    let selectedText;
+    try {
+      selectedText = await getSelectedText();
+    } catch (err) {
+      console.log(err);
+    }
+    if (selectedText) {
+      setSearchText(selectedText);
+    } else {
+      console.log("no selected text");
+    }
+  }
+
+  return (
+    <List
+      filtering={false}
+      isLoading={isLoading}
+      searchText={searchText}
+      onSearchTextChange={setSearchText}
+      navigationTitle="Send commands to Hints AI Assistant"
```

Comment:
Don't change the `navigationTitle` in the root command - it will be automatically set to the command name. Use `navigationTitle` only in nested screens to provide additional context

[Missing an icon](https://developers.raycast.com/api-reference/user-interface/icons-and-images)